resources/views/components/search.blade.php

```diff
@@ -0,0 +1,23 @@
+@props(['search' => ''])
+
+@if (\Firefly\Features::enabled('search'))
+    <form method="GET">
+        <div>
+            <label for="search" class="block text-lg font-medium text-gray-700 hidden">Quick search</label>
```

Comment:
"block" and "hidden" on the same element? Do you mean to remove these and use "sr-only" or something?

Reply:
yes, will update